src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/ProteinsList.scss

```diff
@@ -1,4 +1,13 @@
 .proteinsList {
   font-size: 13px;
   font-weight: 300;
+  overflow: auto;
```

Comment:
why?

Reply:
I was playing around with overflow before the layout update PR got merged and I missed to clean this up.